lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -533,89 +525,6 @@ def test_delete_with_some_state(self):
                                           module_state_key=self.location)
 
 
-class TestOra2ResponsesInstructorTask(TestInstructorTasks):
```

Comment:
This test was defined twice (?!).

Reply:
Maybe it was a merge error, and we took both?